roles/openshift_cloud_provider/tasks/main.yml

```diff
@@ -22,3 +22,6 @@
 
 - include_tasks: vsphere.yml
   when: cloudprovider_is_vsphere | bool
+
+- include_tasks: azure.yml
+  when: cloudprovider_is_azure | bool
```

Comment:
Think we need to add this var to vars/main.yml like https://github.com/kwoodson/openshift-ansible/blob/9c519b67ce78af7ac2e321d1c70e1257a4afc1ed/roles/openshift_cloud_provider/vars/main.yml#L5

Reply:
Yes, thanks.  I didn't understand how that variable was defined.

I'd actually prefer to dynamically include those files instead of checking each.  Let me include others in the discussion.